include/sdf/PrintConfig.hh

```diff
@@ -33,6 +33,16 @@ class SDFORMAT_VISIBLE PrintConfig
   /// \brief Default constructor.
   public: PrintConfig();
 
+  /// \brief Set print config to preserve <include> tags.
+  /// \param[in] _preserve True to preserve <include> tags.
+  /// False to expand included model.
+  public: void SetPreserveIncludes(bool _preserve);
+
+  /// \brief Check if <include> tags are to be preserved or expanded.
+  /// \return True if <include> tags are preserved.
+  /// False if they are to be expanded.
+  public: bool PreserveIncludes() const;
+
   /// \brief Private data pointer.
   IGN_UTILS_IMPL_PTR(dataPtr)
 };
```

Comment:
nit: not really directly related to this PR but it seems that other heather files indent everything inside `namespace sdf` maybe we should take this opportunity to do the same in this file for consistency with the rest?

Reply:
Also it seems like `inline namespace SDF_VERSION_NAMESPACE {` is used for this case.

Reply:
01651adc402caa22d04a718efc5189d3b2a9092b